ui/src/limits/actions/index.ts

```diff
@@ -0,0 +1,36 @@
+// API
+import {getReadWriteLimitsAJAX} from 'src/limits/apis'
+
+// Types
+import {AppState} from 'src/types'
+
+// Actions
+import {notify} from 'src/shared/actions/notifications'
+
+// Constants
+import {readWriteLimitReached} from 'src/shared/copy/notifications'
+
+enum LimitStatus {
+  OK = 'ok',
+  EXCEEDED = 'exceeded',
+}
+
+export const getReadWriteLimits = () => async (
+  dispatch,
+  getState: () => AppState
+) => {
+  try {
+    const {
+      orgs: {org},
+    } = getState()
+
+    const limits = await getReadWriteLimitsAJAX(org.id)
+
+    const readLimit = limits.read.status === LimitStatus.EXCEEDED
+    const writeLimit = limits.write.status === LimitStatus.EXCEEDED
```

Comment:
nit: `readLimit` and `writeLimit` sound like they would be the actual limit like a number. seems like these should be something along the lines of `readLimitReached`

Reply:
👍